assets/index.pug

```diff
@@ -0,0 +1,12 @@
+doctype html
```

Comment:
This is not an SPA but a lib, so this file is not needed

Reply:
Removed